config/config.yaml

```diff
@@ -183,9 +183,9 @@ clouds:
         imageRepo: app-sre/uhc-clusters-service
         azureOperatorsManagedIdentities:
           clusterApiAzure:
-            roleName: Contributor
+            roleName: Azure Red Hat OpenShift Control Plane Operator Role - Dev
           controlPlane:
-            roleName: Contributor
+            roleName: Azure Red Hat OpenShift Cluster API Role - Dev
```

Comment:
I assume we need to leave this at `Contributor` until we make the KMS changes in Hypershift

Reply:
perhaps https://github.com/openshift/hypershift/blob/main/cmd/infra/azure/create.go#L831

how does `Azure Red Hat OpenShift Image Registry Operator - Dev` maps to `Azure Red Hat OpenShift Image Registry Operator Role` ?

Reply:
thanks. Should be a 1:1 copy for the most part, unless anyone wants to change permissions going forward.
In DEV they want to keep the custom roles for this reason.

Reply:
actually, this might get mapped from cluster service to the hypershift API via:
https://github.com/openshift/hypershift/pull/4811/files#diff-11b7c56a5faf0c5e2154fc35474456e2aac1b6ea8e22e5bbda6c3a3996d91afaR423

there is also a KMS field already